### PR TITLE
ruby-build 20150519

### PR DESCRIPTION
**Pull request by mislav**

### CODE REVIEW

File: Library/Formula/ruby-build.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class RubyBuild < Formula
   homepage "https://github.com/sstephenson/ruby-build"
-  url "https://github.com/sstephenson/ruby-build/archive/v20150506.tar.gz"
+  url "https://github.com/sstephenson/ruby-build/archive/v20150519.tar.gz"
   sha256 "68bc8577897f0a7f76b02a7a666d613de9d15379f5b8b7c6f753c75bd9fade22"
```
Comment (DomT4) [inline Library/Formula/ruby-build.rb]: `sha256` needs updating too :wink:

Comment (mislav) [inline Library/Formula/ruby-build.rb]: Updated. Sorry, my release script wasn't aware that homebrew switched to `sha256` in the meantime.

Comment (DomT4) [inline Library/Formula/ruby-build.rb]: Yeah. Mandatory for new formula, very heavily preferred for existing formula on update.

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**